src/pook/pytest_fixture.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+import pook as pook_mod
+
+
+@pytest.fixture
+def pook():
+    """Pytest fixture for HTTP traffic mocking and testing"""
+    pook_mod.on()
+    yield pook_mod
+    pook_mod.off()
```

Comment:
```suggestion
    with pook_mod.use() as engine:
        yield engine
```

Reply:
This would not work with intended usage, which look like [the example included in this PR](https://github.com/wimglenn/pook/blob/db562b235df357e1405ee866334f98ad5473f382/examples/pytest_example.py#L41-L44).

Now it would be something like 

    AttributeError: 'Engine' object has no attribute 'get'

How do you use the engine?  Can the fixture still yield the module instead?

Reply:
>  Can the fixture still yield the module instead?

Yes, I think that's right, apologies!